pkg/controller/controller.go

```diff
@@ -289,3 +313,13 @@ func getFormDetails(r *http.Request) *FormDetails {
 		},
 	}
 }
+
+func (c *Controller) renderBadRequest(w http.ResponseWriter, m string) {
+	t := http.StatusText(http.StatusBadRequest)
+	forbiddenDetails := &ErrorDetails{
+		PageTitle:   t,
+		Description: t,
+		Message:     m,
+	}
+	c.h.RenderHTMLStatus(w, http.StatusBadRequest, "400.html.tmpl", forbiddenDetails)
```

Comment:
You can inline this:

```suggestion
	c.h.RenderHTMLStatus(w, http.StatusBadRequest, "400.html.tmpl", &ErrorDetails{
		PageTitle:   t,
		Description: t,
		Message:     m,
	})
```